test/bigtest/tests/POLine/line-create-test.js

```diff
@@ -49,11 +49,12 @@ describe('Create PO Line simple test', function () {
         await lineEditPage.quantityPhysical.fill(2);
         await lineEditPage.physicalCreateInventory.select('None');
         await lineEditPage.saveButton.click();
-        await orderDetailsPage.whenLoaded();
+        await ordersList.whenListLoaded();
```

Comment:
why orders list? in ticket mentioned that line details pane should be opened

Reply:
Form is closed and OrdersList with Line details are presented

Reply:
so what is the point of this test if it doesn't test ticket's requirement?

Reply:
Appreciate your support with test